contexts/MembershipContext/src/DataAccess/DoctrineApplicationRepository.php

```diff
@@ -142,13 +169,17 @@ private function getDoctrineStatus( Application $application ): int {
 			$status += DoctrineApplication::STATUS_CANCELED;
 		}
 
-		if ( $status === DoctrineApplication::STATUS_NEUTRAL ) {
+		if ( $status === DoctrineApplication::STATUS_NEUTRAL && $this->isDirectDebitPayment( $application ) ) {
 			$status += DoctrineApplication::STATUS_CONFIRMED;
```

Comment:
Direct debits were first persisted with a neutral status and updated to confirmed once the SEPA checkbox form was sent.